05/app.js

```diff
@@ -9,27 +9,41 @@ const stats = {
 
 /* tutaj umieść swój kod */
 
+const textClick = document.querySelectorAll('p');
 
+const update = function (e) {
+    if (e.target.getAttribute('href')) {
+        e.preventDefault();
+        const name = e.target.getAttribute('href');
+        typeof stats.links[name] === 'undefined' ? stats.links[name] = 1 : stats.links[name]++;
+    } else if (e.target.dataset.id) {
+        stats.paragraphs.p1++;
```

Comment:
To rozwiązanie działa tylko dla `p1`, a mamy jeszcze `p2` i `p3` - zrób podobnie jak dla linków ;)